sentry/migrations/20190806011140_initial-tables/up.sql

```diff
@@ -28,8 +28,19 @@ CREATE INDEX idx_validator_messages_msg_state_root ON validator_messages ((msg -
 CREATE TABLE event_aggregates
 (
     channel_id VARCHAR(66)              NOT NULL REFERENCES channels (id) ON DELETE RESTRICT,
-    created    TIMESTAMP WITH TIME ZONE NOT NULL,
-    events     JSONB                    NOT NULL
+    created    TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    event_type VARCHAR(255)             NOT NULL,
+    earner     VARCHAR(255),
+    event_counts VARCHAR                   NOT NULL,
```

Comment:
can't we use the native bigint here? also, `count` and `payout` is sufficient in terms of naming

Reply:
We can't use BIGINT because it can only represent 64 bits integers while we need 256 bits integer representation